src/coffee/bootstrap.coffee

```diff
@@ -4,5 +4,9 @@ define deps, (require, ng) ->
   # The call to setTimeout is here as it makes loading the app considerably more reliable.
   # Depending on compilation sequence, various modules were not being found. This is dumb, and
   # a better way ought to be found.
-  require ['domReady!'], (document) -> setTimeout (-> ng.bootstrap document, ['steps']), 100
+  require ['domReady!'], (document) ->
```

Comment:
It would probably be cleaner, if it cannot be declared inline, to extract the actual bootstrapping as a named function. Btw: what does the event.props thing do?